src/hooks/useGridDimensions.ts

```diff
@@ -17,11 +18,13 @@ export function useGridDimensions() {
 
     const { clientWidth, clientHeight, offsetWidth, offsetHeight } = gridRef.current!;
     const { width, height } = gridRef.current!.getBoundingClientRect();
-    const initialWidth = width - offsetWidth + clientWidth;
+    const horizontalScrollbarWidth = offsetWidth - clientWidth;
```

Comment:
Isn't that the vertical scrollbar width instead?

Reply:
🤦‍♂️